### ISSUE TITLE
Support Python 3

### ISSUE THREAD

**Issue by pombredanne** [sassoftware/pymaven#14]
We are working on the Python3 port of Scancode and this library will need to be updated accordingly to support Python3.

For now there are issues such as https://github.com/sassoftware/pymaven/blob/b744bbe27d6418ee169c8f8718ce89f602e21b2a/pymaven/utils.py#L19
```
    from urlparse import urlsplit, urlunsplit
ModuleNotFoundError: No module named 'urlparse'
```

**Comment by pombredanne** [comment]
@wfscheper It looks like scancode and your tools are the only two users of pymaven so far. Are you still willing to maintain it? Otherwise I am fine ti take it over. Alternatively I can just maintain my fork too.

**Comment by wfscheper** [comment]
Whoops. Hit the wrong button. I'm taking at look what it will take to add python3 support.

**Comment by wfscheper** [comment]
So PR #13 adds python3 support, but I left it un-merged for you to review. Guess we both got busy with other things. Shall I merge that now?

**Comment by pombredanne** [comment]
@wfscheper let me quickly review this at last!

**Comment by wfscheper** [comment]
Fixed by #13
